scout/adapter/mongo/clinvar.py

```diff
@@ -262,7 +262,7 @@ def _basic_submission_info(self, result: dict) -> dict:
             "updated_at": result.get("updated_at"),
         }
 
-    def get_clinvar_onc_submissions(self, institute_id: str) -> pymongo.synchronous.cursor.Cursor:
+    def get_clinvar_onc_submissions(self, institute_id: str) -> pymongo.cursor.Cursor:
```

Comment:
💯 I never noticed you merged it anyway - thought I mentioned it. 😁

Reply:
Ok! I didn't realise you saw the error, I thought it was more of a comment!

Reply:
Well, I don't want to be the one who admits to wanting to I-told-you-so, but it was a code suggestion and a reminder it crashed the last time. 😂